Makefile

```diff
@@ -16,6 +16,12 @@ lint:
 	@./lint.sh
 test:
 	bazel test --test_output=all //lib/...
+image-e2e-env:
+	bazel build $(BAZEL_BUILD_OPTS) //:e2e-env-docker-loadable.tar
+image-load-e2e-env: image-e2e-env
+	docker load -i bazel-bin/e2e-env-docker-loadable.tar
```

Comment:
I thought there was a bazel command to do this?  `bazel run //:e2e-env-docker-loadable`